### PR TITLE
Flexbox homework

### PR DESCRIPTION
**Pull request by paristechprofile**
note: need a bit of help solidifying sibling relative relationships.

### CODE REVIEW

**Comment by DaltonHart** [comment]
Thank you for having an organized commit history. Remember to make your commit messages as easy to follow as possible.  For example, using "pushing homework" isn't explaining what is the difference in your code between commits. A better example would be "added Js racing game code completed" now ideally you should have commits throughout your development cycle. If you follow this format you will save yourself (and your codevs) a lot of  headache 😄

File: paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    },
+  };
+};
+
+//activate button listening
+const go = document.querySelector(".ready").addEventListener("click", function() {
+//show a user a GO!
+    document.getElementsByTagName("h2")[0].innerText = "GO!";
+});
+
```
Comment (DaltonHart) [inline paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js]: I like the use of const in your document, but I can tell that this code is a majority of the solution code. I would like it when you have time to go back and work through the logic of the game differently. I can tell from some of your other code that you can produce more advanced logic. Would love to see your style shine. ⭐️

File: paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	var blueBoxRight = parseInt(document.querySelector('.blueBox').getBoundingClientRect().right);
+//determine redBox right boundary
+	var redBoxRight = parseInt(document.querySelector('.redBox').getBoundingClientRect().right);
+//determine finish line 
+	let finishLine = parseInt(document.querySelector('.finish').getBoundingClientRect().left);
+
+let blueBox = document.querySelector('.blueBox');
+let redBox =  document.querySelector('.redBox') //do the same for red
+  	window.addEventListener('keydown', moveBox); // this is comprised of an event and a callback function 
+  	var blueLeft = 0;
```
Comment (DaltonHart) [inline paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js]: If you are using Let in your document you should not be using var as well. It is not a good practice to mix the two. Moving forward try to stick to one or the other.

**Comment by paristechprofile** [comment]
Context: this assignment was pretty difficult because I don't have previous
experience coding. But, I'm definitely open to the challenge! Can you give
me a more specific challenge in person or via email (e.g. use x concept to
achieve y result)? The specifics will help me cement my understanding and
do so with a bit more pizzazz.

**Comment by paristechprofile** [comment]
Roger that! Thank you. Really loving the feedback

**Pull request merged**